project-base/storefront/tailwind.config.js

```diff
@@ -14,36 +14,28 @@ module.exports = {
             xxl: em(1560),
```

Comment:
Apply promocode button is broken 
<img width="265" alt="image" src="https://github.com/shopsys/shopsys/assets/55181354/18b88ec6-0236-4615-9c65-b79f84076d87">

Reply:
I will resolve it here, let's handle it somewhere else